docs/reference/analysis/tokenfilters/synonym-graph-tokenfilter.asciidoc

```diff
@@ -150,8 +150,8 @@ To apply synonyms, you will need to include a synonym graph token filter into an
 Order is important for your token filters.
 Text will be processed first through filters preceding the synonym filter before being processed by the synonym filter.
 
-In the above example, text will be lowercased by the `lowercase` filter before being processed by the `synonyms_filter`.
-This means that all the synonyms defined there needs to be in lowercase, or they won't be found by the synonyms filter.
+{es} will also use the token filters preceding the synonym filter in a tokenizer chain to parse the entries in a synonym file.
+In the above example, the synonyms graph token filter is placed after a stemmer. The stemmer will also be applied to the synonym entries.
```

Comment:
@mayya-sharipova its important to clarify if we also apply the token filters on the synonyms when they are loaded from the file. This way any stemmer is also applied to the previously stored synonyms.

Reply:
I've tried to make the example as close as possible to the previous one to avoid further confusion. I've added synonym set as well to synonym file to ensure users understand that it applies to any synonym configuration.